hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -496,11 +497,17 @@ public void importContainerData(InputStream input,
       KeyValueContainerUtil.parseKVContainerData(containerData, config);
 
     } catch (Exception ex) {
+      if (ex instanceof StorageContainerException &&
```

Comment:
NIT. Following may be better to handle the exception cases.

```
} catch (StorageContainerException ex) {
  throw ex;
} catch (Exception ex) {
  //delete all the temporary data in case of any exception.
  ...
}
```